src/tests/unit/tests/electron/flux/store/window-state-store.test.ts

```diff
@@ -1,10 +1,10 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
-import { RoutePayload } from 'electron/flux/action/route-payloads';
-import { WindowStateActions } from 'electron/flux/action/window-state-actions';
-import { WindowStateStore } from 'electron/flux/store/window-state-store';
-import { WindowStateStoreData } from 'electron/flux/types/window-state-store-data';
-
+import { WindowStatePayload } from 'electron/flux/action/window-state-payload';
+import { RoutePayload } from '../../../../../../electron/flux/action/route-payloads';
```

Comment:
all imports from the `src/electron` folder should be non-relative (as before the changes)
We change a setting to make this easier for everyone under `.vscode\settings.json`, property `    "typescript.preferences.importModuleSpecifier": "non-relative"`; you can check if you already had that or update from master maybe?

Reply:
I think this might have been a remnant from a merge from master.